pages/documentation/index.html

```diff
@@ -362,6 +362,20 @@ <h5>
             </div>
         </div>
     </div>
+
+    <!-- Remedial Action Optimizer -->
+    <div class="col-lg-4 col-sm-6 my-2">
+        <div class="media documentation_item">
+            <div class="media-body">
+                <h5>
+                    <a class="documentation_title" href="simulation/rao">
+                        <em class="doc-icon align-self-start bx bxs-shield"></em>Remedial Action Optimizer
+                    </a>
+                </h5>
+                <div class="documentation_descr">Optimize remedial actions in the network in order to relieve congestions. </div>
```

Comment:
congestions is not really used in the powsybl world. We prefer talking about operational limit violations. But if it is widely used in open rao, maybe add a small definition in the page below.